hook.py

```diff
@@ -77,6 +77,8 @@ def check_if_record_is_deployed(domain, dns_record, token):
             logger.info(" + Record not available yet. Checking again in 10s...")
         except dns.exception.Timeout:
             logger.info(" + DNS Request timeout. Checking again in 10s...")
+        except dns.resolver.NoAnswer:
```

Comment:
Coud you use the except statement just below to regroup same message exceptions ?

like this

```python
except (dns.resolver.NoAnswer, dns.resolver.NXDOMAIN):
```